src/Entity/User.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\UserRepository")
+ */
+class User
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $username;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $password;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $role;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $seniority;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=30)
+     */
+    private $surname;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $createdAt;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $updatedAt;
+
+    /**
+     * @var \Doctrine\Common\Collections\Collection|Technology[]
```

Comment:
Import `\Doctrine\Common\Collections\Collection` in the `use` block so here you will have `Collection|Technology[]`